draft-thomson-http-oblivious.md

```diff
@@ -916,6 +916,50 @@ corresponds to the key identifier, but the encapsulated request cannot be
 successfully decrypted using the key.
 
 
+## Replay Attacks
+
+Encapsulated requests can be copied and replayed toward servers. The replay
+risk is similar to the exposure to replay attack in TLS early data (see
+{{!TLS=RFC8446}}).
```

Comment:
```suggestion
{{!TLS=RFC8446}}). However, one important difference is that the oblivious proxy, which handles
encapsulated requests, can replay messages; replay attacks by network adversaries are less 
applicable unless clients or proxies send encapsulated requests as early data. 
```

Reply:
... basically, trying to say that we limit the attack surface here compared to TLS, in that only proxies can reasonably replay, compared to general network attackers.

Reply:
Are we saying here that we don't necessarily trust the proxy not to replay?  I like the observation about replay exposure to TLS early data.  I think that both work, but I want to try capturing this in a different way.

Reply:
Dunno, we might not 🤷  I could imagine a proxy attempting to be "helpful" and replaying messages on a client's behalf.

Reply:
It's potentially a nice feature of a proxy (though I suspect most won't want to hold the necessary state).

Reply:
Yeah, my comment was more to note that this type of replay is much easier to do by a proxy, if it was going to do it. (Less so of the network attacker, which is assumed malicious.)